lib/AST/IRToASTVisitor.cpp

```diff
@@ -61,9 +63,86 @@ class ExprGen : public llvm::InstVisitor<ExprGen, clang::Expr *> {
   clang::Expr *visitUnaryOperator(llvm::UnaryOperator &inst);
 };
 
+clang::Expr *IRToASTVisitor::ConvertExpr(z3::expr expr) {
```

Comment:
Can we add short comments on what cases each piece of code handles here?

Reply:
Added some clarifications